setup.py

```diff
@@ -48,6 +48,42 @@
 It should work without additional code in any environment with
 Python >= 2.4, ctypes and an pre-built usb backend library
 (currently, libusb 0.1.x, libusb 1.x, and OpenUSB).
-"""
+""",
+    classifiers=[
+        'Development Status :: 4 - Beta',
+        'Intended Audience :: Developers',
+        'Intended Audience :: Information Technology',
+        'Intended Audience :: Manufacturing', # USB automation, or mfg USB devs
+        'Intended Audience :: Science/Research', # interface with instruments
+        'Intended Audience :: System Administrators', # integrate strange devs
+        'Intended Audience :: Telecommunications Industry', # telecomm devs
+        'License :: OSI Approved :: BSD License',
+        'Natural Language :: English',
+        # try to union the OSes that can build any of the backend libraries...
+        'Operating System :: MacOS :: MacOS X',
+        'Operating System :: Microsoft :: Windows :: Windows Vista',
+        'Operating System :: Microsoft :: Windows :: Windows 7',
+        'Operating System :: POSIX :: BSD :: FreeBSD',
+        'Operating System :: POSIX :: BSD :: NetBSD',
+        'Operating System :: POSIX :: BSD :: OpenBSD',
+        'Operating System :: POSIX :: Linux',
+        'Operating System :: POSIX :: SunOS/Solaris',
+        'Programming Language :: Python :: 2.4',
+        'Programming Language :: Python :: 2.5',
+        'Programming Language :: Python :: 2.6',
+        'Programming Language :: Python :: 2.7',
+        'Programming Language :: Python :: 3',
+        # source(CPython,Jython,IronPython,PyPy): "The Long Term" section of
+        # http://ojs.pythonpapers.org/index.php/tpp/article/viewFile/23/23
+        'Programming Language :: Python :: Implementation :: CPython',
+        'Programming Language :: Python :: Implementation :: IronPython',
+        'Programming Language :: Python :: Implementation :: Jython',
+        'Programming Language :: Python :: Implementation :: PyPy',
+        'Topic :: Scientific/Engineering :' \
+            ': Interface Engine/Protocol Translator',
```

Comment:
I can see this'll come down to whether you hate having it line-broken more than I hate making >80-column text files. :) Maybe we can sell tickets as a fundraiser....